xcode/Subconscious/Shared/Components/FirstRun/FirstRunView.swift

```diff
@@ -46,26 +46,62 @@ struct FirstRunView: View {
                 
                 Spacer()
                 
-                ValidatedFormField(
-                    alignment: .center,
-                    placeholder: "Enter your invite code",
-                    field: app.state.inviteCodeFormField,
-                    send: Address.forward(
-                        send: app.send,
-                        tag: AppAction.inviteCodeFormField
-                    ),
-                    caption: "Look for this in your welcome email.",
-                    submitLabel: .go,
-                    onSubmit: {
-                        if app.state.inviteCodeFormField.isValid {
-                            app.send(.pushFirstRunStep(.nickname))
+                if let gatewayId = app.state.gatewayId {
+                    VStack(spacing: AppTheme.unit2) {
+                        HStack(spacing: AppTheme.unit2) {
+                            Text("Invite code redeemed")
+                        Image(systemName: "checkmark.circle")
+                            .resizable()
+                            .frame(width: 16, height: 16)
+                            .opacity(0.5)
                         }
+                        .font(.body)
+                        .bold()
+                        
+                        HStack(spacing: AppTheme.unit) {
+                            Text("Gateway ID")
+                                .bold()
+                            Text(gatewayId)
+                        }
+                        .font(.caption)
                     }
-                )
-                .textFieldStyle(.roundedBorder)
-                .textInputAutocapitalization(.never)
-                .disableAutocorrection(true)
-                
+                    .foregroundColor(.secondary)
```

Comment:
On un-focusing the field we will attempt to redeem the invite code and, if successful, we replace the text field entirely by a message:

<img width="358" alt="image" src="https://github.com/subconsciousnetwork/subconscious/assets/5009316/55e34dc0-504e-468b-bfff-d6bdfc833ac1">

This is a quick pass visually, open to feedback / refinement.

Reply:
Here's the error state:

<img width="358" alt="image" src="https://github.com/subconsciousnetwork/subconscious/assets/5009316/b9c26951-2c3d-4e95-a7e2-0355d782b7ea">

Reply:
WFM